app/main.js

```diff
@@ -94,6 +96,22 @@ app.on('before-quit', () => {
 })
 
```

Comment:
We should remove the default window-close-quitting behavior by adding a line to listen on the `window-all-closed` event:

```suggestion
app.on('window-all-closed', () => {})
```